provider/vpc/resource/network.py

```diff
@@ -1,373 +1,585 @@
 from typing import List
 
 from provider.vpc.command import VpcOptions
-from shared.common import *
+from shared.common import (
+    ResourceProvider,
+    Resource,
+    message_handler,
+    get_name_tags,
+    ResourceDigest,
+    ResourceEdge,
+)
 from shared.error_handler import exception
 
 
-class INTERNETGATEWAY(object):
-
+class INTERNETGATEWAY(ResourceProvider):
     def __init__(self, vpc_options: VpcOptions):
+        """
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [One-line docstring should fit on one line with quotes (found 4) (D200)](https://app.codacy.com/gh/Cloud-Architects/cloud-discovery/pullRequest?prid=5655786)